docs: Fix install grammar (#6118)
mealie-recipes/mealie: docs/docs/documentation/getting-started/installation/installation-checklist.md

## Step 3: Customizing The `docker-compose.yaml` files.

> After you've decided how to setup the files it's important to set a few ENV variables to ensure that you can use all the features of Mealie. I recommend that you verify and check that:

Reviewer: We can probably make this a little cleaner still:

```suggestion
After you've decided how to set up your files, it's important to set a few ENV variables to ensure that you can use all the features of Mealie. Verify that:
```
